code/modules/organs/ailments/ailment_codex.dm

```diff
@@ -39,7 +39,7 @@
 	ailment_table += "<tr><td><b>[name_column]</b></td><td><b>[treatment_column]</b></td></tr>"
 	for(var/atype in subtypesof(/datum/ailment))
 		var/datum/ailment/ailment = get_ailment_reference(atype)
-		if(!ailment.name || show_robotics_recipes != ailment.affects_robotics || ailment.hidden_from_codex)
+		if(!ailment.name || show_robotics_recipes != ailment.applies_to_prosthetics || ailment.hidden_from_codex)
```

Comment:
shouldn't it be `applies_to_robotics`, since the faults are for robotic prosthetic limbs? the robotic prosthetic faults guide even talks about EMPs and stuff like that.

Reply:
i guess, that works